### DISCUSSION TITLE
MAX_SITEMAPS_SEEN as parameter on sitemap_search

### DISCUSSION THREAD

**Discussion by felipehertzer** [adbar/trafilatura#686] [Ideas]
Hey @adbar,

I've been testing the sitemap function and I was wondering if there’s an easier way to limit the maximum URLs retrieved by the search other than using MAX_SITEMAPS_SEEN. I noticed this setting is in the settings rather than as a parameter. Do you think it would be possible to move it as a parameter?

Thanks.

**Comment by adbar** [comment]
Yes, `MAX_SITEMAPS_SEEN` can remain the default value but a user-defined parameter can by passed to `sitemap_search()` and stored in the `SitemapObject` class.

Note that `settings.cfg` is also an option but I wonder who uses this file, a function parameter is probably more convenient.

Feel free to draft a PR!
